web/src/copy_and_paste.js

```diff
@@ -341,7 +346,29 @@ export function paste_handler(event) {
     }
 
     if (clipboardData.getData) {
+        console.log(event);
```

Comment:
The linters will certainly prompt us to delete this, but consider this a friendly reminder. ;)